Temperature_API/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime)
+    temperature = Column(Integer, nullable=False)
+    city_id = Column(Integer, ForeignKey("city.id"))
```

Comment:
Ensure that the `city_id` column is correctly linked to the `City` model's primary key. Consider setting `nullable=False` if every temperature record must be associated with a city.